src/library-authoring/index.ts

```diff
@@ -1,2 +1,4 @@
-export { default as LibraryAuthoringPage } from './LibraryAuthoringPage';
+// @ts-check
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
We can remove this
```suggestion
```

Reply:
Updated https://github.com/open-craft/frontend-app-course-authoring/pull/38/commits/d37ffbf4000b14cfe5ea3065056e7c3d1da1f4e5